### ISSUE TITLE
NTR: Soluble CD163 high in blood (Lab: serology & immunological tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8752]
New term request Soluble CD163 high in blood (ang_130706201114): SCD163 raised; Cluster of Differentiation 163

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000385
- label: Elevated circulating soluble CD163 concentration
- parents: [Abnormal circulating protein concentration (HP:0010876)](https://hpo.jax.org/app/browse/term/HP:0010876)
- pmid: PMID:21713198
- def: The concentration of Soluble CD163 in the blood circulation is above the upper limit of normal.
- comment: Soluble CD163 is considered to be a marker of alternatively activated (anti-inflammatory) macrophages.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8752
- synonyms:
